po/nl.po

```diff
@@ -0,0 +1,140 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the platecodes.arc676 package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: platecodes.arc676\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-09-05 16:01+0000\n"
+"PO-Revision-Date: 2021-09-23 18:27+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.0\n"
+"Last-Translator: Heimen Stoffels <vistausss@fastmail.com>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: nl\n"
+
+#: ../qml/Main.qml:45
+msgid "Regex:"
+msgstr "Reguliere uitdrukking:"
+
+#: ../qml/Main.qml:92
+msgid "Embassies in Beijing, China"
+msgstr "Ambassades in Peking, China"
+
+#: ../qml/Main.qml:93
+msgid "Chinese Provinces"
+msgstr "Chinese provincies"
+
+#: ../qml/Main.qml:94
+msgid "Italian Provinces"
+msgstr "Italiaanse privoncies"
```

Comment:
Thank you for your contribution! Is it intentional that "provinces" is translated differently for these two strings?

Reply:
No, that's a typo. Thanks for catching it - I corrected it now. :)